src/v2/stack_aggregator.py

```diff
@@ -398,15 +399,18 @@ def create_result(self, **kwargs) -> StackAggregatorResultForRegisteredUser:
 
 def initiate_unknown_package_ingestion(aggregator: Aggregator):
     """Ingestion of Unknown dependencies."""
-    ecosystem = aggregator._normalized_packages.ecosystem
-    try:
-        for dep in aggregator.get_all_unknown_packages():
-            server_create_analysis(ecosystem, dep.name, dep.version, api_flow=True,
-                                   force=False, force_graph_sync=True)
-    except Exception as e:  # pylint:disable=W0703,C0103
-        logger.error('Ingestion failed for {%s, %s, %s}',
-                     ecosystem, dep.name, dep.version)
-        logger.error(e)
+    if os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW", "") == "1":
```

Comment:
Don't use os.environ directly, Refer settings.py and add an entry to disable unknown ingestion. Also do the same for v1 as well.